q2_alignment/__init__.py

```diff
@@ -10,7 +10,10 @@
 
 from ._mafft import mafft
 from ._filter import mask
+from ._version import get_versions
 
-__version__ = pkg_resources.get_distribution('q2-alignment').version
```

Comment:
Can you remove `import pkg_resources` above? I think flake8 will complain otherwise.